### PR TITLE
CI/CD, orbs, workflows

### PR DESCRIPTION
**Pull request by JosueMolinaMorales**
This PR partially completes story [7391](https://app.shortcut.com/clarkcan/story/7391/ops-course-circleci-lesson-documentation)

### CODE REVIEW

File: Ops/Ops-lessons-2022/CircleCI.md [hunk 1]
```diff
@@ -28,6 +38,34 @@
 
 ### Orbs
 
+Orbs are reusable snippers of code that help automate repeated processes, accelerate project setup, and make it easy to integrate with third-party tools such as Node, Docker, and AWS.
```
Comment (pzalep1) [inline Ops/Ops-lessons-2022/CircleCI.md]:
```suggestion
Orbs are reusable snippets of code that help automate repeated processes, accelerate project setup, and make it easy to integrate with third-party tools such as Node, Docker, and AWS. 
```

Comment (pzalep1) [reply Ops/Ops-lessons-2022/CircleCI.md]: Not sure if this is what you meant

File: Ops/Ops-lessons-2022/CircleCI.md [hunk 2]
```diff
@@ -28,6 +38,34 @@
...
+  
+  workflows:
+    maven_test:
+      jobs:
+        - maven/test
+```
+
+In this case, ``` maven: circleci/maven@0.0.12 ```, maven can be thought as the object reference variable and is used in the workflow named ``` maven_test ``` 
+
+You can create your own orb and use them within your organization only or post it on the circleci orb registry
```
Comment (pzalep1) [inline Ops/Ops-lessons-2022/CircleCI.md]:
```suggestion
You can create your own orb and use them within your organization only or post it on the circleci orb registry.
```

File: Ops/Ops-lessons-2022/CircleCI.md [hunk 3]
```diff
@@ -28,6 +38,34 @@
...
+    maven_test:
+      jobs:
+        - maven/test
+```
+
+In this case, ``` maven: circleci/maven@0.0.12 ```, maven can be thought as the object reference variable and is used in the workflow named ``` maven_test ``` 
+
+You can create your own orb and use them within your organization only or post it on the circleci orb registry.
+
+SecurEd will be creating their own orb to make our config.yml shorters and abstract many jobs and commands to a single repository.
```
Comment (EVogel1999) [inline Ops/Ops-lessons-2022/CircleCI.md:61]: Not for this PR but when we make this we should add a link to this playbook to the repo.

**Review by StarChaser44 — APPROVED** [review]

**Review by EVogel1999 — APPROVED** [review]
Looks good!  Just one comment but nothing to change.  Just something to keep in mind once we have a company orb.

File: Ops/Ops-lessons-2022/CircleCI.md (hunk 3 above)
Comment (JosueMolinaMorales) [reply Ops/Ops-lessons-2022/CircleCI.md:61]: Agreed

**Review by pzalep1 — APPROVED** [review]

**Review by mfranz1 — APPROVED** [review]

**Review by Cwagne17 — APPROVED** [review]

**Pull request merged**